### PR TITLE
closes Issue12 and closes issue17

### LINKED ISSUE #12: Update Culture doc
Using the skeleton layout, provide details and content to each section laid out in content/culture.md

### LINKED ISSUE #17: Link markdown files for github pages for first preview
Adding markdown link sections to create a single document to publish initial versions of content

### PR DESCRIPTION
**Pull request by killumslow**
Latest updates for the culture section and included placeholder links to navigate sections  in the documents

### CODE REVIEW

File: content/culture.md [hunk 1]
```diff
@@ -1,12 +1,64 @@
...
+  - merit based reward system
 
+How corporate culture affects social culture.
+
+What is valued?
+  - employees
+  - ideas/innovation
+  - work output
+How are issues resolved/handled?
+  - isolated and quiet
```
Comment (skypatrolcpt) [inline content/culture.md]: "Isolated and quiet" and "free to broadcast concerns" could be conflict with each other

**Review by skypatrolcpt — COMMENTED** [review]
One comment on culture, but could just be me reading the doc out of proper context.

File: content/culture.md [hunk 2]
```diff
@@ -1,8 +1,34 @@
...
+## Management communication approaches that affect diversity and inclusion
+
+Open door policies
+
+Tell HR
+
+Up to supervisor or manager to handle
 
 
 ## ways to establish and maintain organizational culture
```
Comment (bgordon134) [inline content/culture.md]: Capitalize "Ways"

File: content/culture.md (hunk 1 above)
Comment (bgordon134) [reply content/culture.md]: Maybe we clarify that it's based on individual preference. Some people may want to have an offline conversation. Others are comfortable starting a wider discussion.

File: content/culture.md [hunk 3]
```diff
@@ -1,8 +1,34 @@
 # Culture
 
-## implementation methods that affect diversity and inclusion
+High level description of corporate culture what it means and its importance.
 
-Methodologies
+focus on the primarily used corporate and organizational culture
```
Comment (bgordon134) [inline content/culture.md]: Let's capitalize this, too.

File: content/culture.md [hunk 4]
```diff
@@ -1,8 +1,34 @@
...
+- Heiarchical
```
Comment (bgordon134) [inline content/culture.md]: Hierarchical

File: content/culture.md [hunk 5]
```diff
@@ -1,8 +1,34 @@
...
+  - position/senority based
```
Comment (bgordon134) [inline content/culture.md]: seniority

File: content/culture.md [hunk 6]
```diff
@@ -1,8 +1,34 @@
...
+  - are they tracked to identify large scale problems
+  - just enough to be compliant and above board legally
+
+## Management communication approaches that affect diversity and inclusion
+
+Open door policies
+
+Tell HR
+
+Up to supervisor or manager to handle
```
Comment (bgordon134) [inline content/culture.md:32]: We probably need to noodle on some talking points for supervisors and managers at some point. They are likely going to be a first line of defense.

**Review by bgordon134 — APPROVED** [review]

File: content/culture.md [hunk 7]
```diff
@@ -50,6 +50,12 @@ Protect diversity
 
 ## ways to establish and maintain organizational culture
 
+1. evalute current culture
+2. create transition plan to accomodate D&I initiatives
+3. transition
+4. post transition engagement
+5. create reevulation methods to identify problems and succsesses
```
Comment (bgordon134) [inline content/culture.md]: create **reevaluation** methods to identify problems and **successes**

File: content/culture.md [hunk 8]
```diff
@@ -50,6 +50,12 @@ Protect diversity
 
 ## ways to establish and maintain organizational culture
 
+1. evalute current culture
+2. create transition plan to accomodate D&I initiatives
+3. transition
```
Comment (bgordon134) [inline content/culture.md]: For this one, should we consider something like, "execute transition plan"?

File: content/culture.md [hunk 9]
```diff
@@ -50,6 +50,12 @@ Protect diversity
...
+4. post transition engagement
+5. create reevulation methods to identify problems and succsesses
+
```
Comment (bgordon134) [inline content/culture.md:60]: 6. feedback gap areas/outcomes into new transition plan and execution

File: content/culture.md (hunk 1 above)
Comment (killumslow) [reply content/culture.md]: These would be examples of conflict resolution. I think both have pros and cons that should be discussed in further depending on culture and individuals.

File: content/culture.md (hunk 2 above)
Comment (killumslow) [reply content/culture.md]: done

File: content/culture.md (hunk 6 above)
Comment (killumslow) [reply content/culture.md:32]: Agreed and noted

File: content/culture.md (hunk 7 above)
Comment (killumslow) [reply content/culture.md]: done

**Pull request merged**